reference/monitoring.md

```diff
@@ -0,0 +1,74 @@
+# Monitoring
+
+## Pool metrics exporter
+
+The Mayastor pool metrics exporter runs as a sidecar container within every io-engine pod and exposes pool usage metrics in Prometheus format. These metrics are exposed on port 9502 using an HTTP endpoint /metrics and are refreshed every five minutes.
+
+### Supported metrics
+
+| Name | Type | Unit | Description |
+| :--- | :--- | :--- | :--- |
+| disk_pool_total_size_bytes | Gauge | Integer | Total size of the pool |
+| disk_pool_used_size_bytes | Gauge | Integer | Used size of the pool |
+| disk_pool_status | Gauge | Integer | Status of the pool (0, 1, 2, 3) = {"Unknown", "Online", "Degraded", "Faulted"} |
+
+
+{% tab title="Example metrics" %}
+```text
+# HELP disk_pool_status mayastor name status
+# TYPE disk_pool_status gauge
+disk_pool_status{node="worker-0",name="mayastor-disk-pool"} 1
+# HELP disk_pool_total_size_bytes mayastor name total size in bytes
+# TYPE disk_pool_total_size_bytes gauge
+disk_pool_total_size_bytes{node="worker-0",name="mayastor-disk-pool"} 5.360320512e+09
+# HELP disk_pool_used_size_bytes mayastor name used size in bytes
```

Comment:
What does "mayastor name" imply? Did you mean disk pool name?

Reply:
I have taken this example output from here: [link](https://dcsw-bolt.atlassian.net/wiki/spaces/MS/pages/4919214/Pool+metrics+exporter#Example-of-the-above-mentioned-metrics%3A)

Reply:
Can you once check for cStor or any other engine of OpenEBS?

Reply:
updated the text.